codegen/java/src/main/kotlin/io/spine/protodata/codegen/java/JavaRejectionConvention.kt

```diff
@@ -35,6 +35,9 @@ import io.spine.tools.code.Language
 
 /**
  * A convention which governs Java Rejection-Throwable class declarations.
+ *
+ * The convention only defines a declaration for rejection message types. Any other types are
+ * undefined and thus result in the [declarationFor] method returning `null`.
  */
 public class JavaRejectionConvention(
```

Comment:
What if we rename this class to `RejectionThrowableConvention`? We have already `java` in the package.

Also, our rejections are two-headed. One "head" is a message, another extends `RejectionThrowable`. Having a more precise name for this class would help to understand its purpose a bit faster.

I am going to grab this class and move to `mc-java` soon. I tried it but couldn't do it because some extension functions were `internal`.

Reply:
Also, I had a comment regarding the name `GeneratedDeclaration`, especially in the context of this class.

The convention for `RejectionThrowable` codegen does not refer to already generated class. We are going to generate this class having this convention.

Therefore, I think that the `Generated` prefix is redundant because it refers to a stage of source code processing, which does not apply in all cases.